doc/source/ray-air/examples/lightgbm_example.ipynb

```diff
@@ -124,10 +121,11 @@
     "\n",
     "    # Scale some random columns, and categorify the categorical_column,\n",
     "    # allowing LightGBM to use its built-in categorical feature support\n",
-    "    preprocessor = Chain(\n",
-    "        Categorizer([\"categorical_column\"]), \n",
-    "        StandardScaler(columns=[\"mean radius\", \"mean texture\"])\n",
-    "    )\n",
+    "    scaler = StandardScaler(columns=[\"mean radius\", \"mean texture\"])\n",
+    "    categorizer = Categorizer([\"categorical_column\"])\n",
+    "\n",
+    "    train_dataset = categorizer.fit_transform(scaler.fit_transform(train_dataset))\n",
```

Comment:
This will cause two passes over the dataset (twice as slow).

Reply:
Discussed offline - this is the same as what `Chain` does.